recipes/couchdb/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{%set name = "CouchDB" %}
+{%set version = "1.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "9503ea413956c33b5e2af248235afb5e0f011f33f816a32e872f8ff9b0a0e710" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
This probably needs a Python prefix as this is merely a Python API to the CouchDB server.